src/components/ShopListItem/ShoptListItem.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Istem1 from "../../images/Istem1";
+
+const ShopListItem = (ShopItemList) => {
+    ShopItemList = {
+        id: 1,
+        img: <Istem1/>, 
```

Comment:
the image should be a `string` instead of `<Image />` because later on the images will come as a `url`
```suggestion
        img: "www.images.com/somerandomimage.jpg", 
```